src/DataDictionary/graph/OverlayPropertyTable/OverlayPropertyTable.jsx

```diff
@@ -1,19 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '@gen3/ui-component/dist/components/Button';
-import { downloadTemplate } from '../../utils';
+import { downloadTemplate, SearchResultItemShape } from '../../utils';
 import { getCategoryIconSVG } from '../../NodeCategories/helper';
+import {
+  getNodeDescriptionFragment,
+  getNodeTitleFragment,
+} from '../../highlightHelper';
 import DataDictionaryPropertyTable from '../../table/DataDictionaryPropertyTable/.';
 import './OverlayPropertyTable.css';
 
 class OverlayPropertyTable extends React.Component {
+  getTitle = () => {
+    if (this.props.isSearchMode) {
+      const nodeTitleFragment = getNodeTitleFragment(
+        this.props.matchedResult.matches,
+        this.props.node.title,
+        'overlay-property-table__span',
+      );
+      return nodeTitleFragment;
+    }
+
+    return this.props.node.title;
+  };
+
+  getDescription = () => {
+    if (this.props.isSearchMode) {
+      const nodeDescriptionFragment = getNodeDescriptionFragment(
+        this.props.matchedResult.matches,
+        this.props.node.description,
+        'overlay-property-table__span',
+      );
+      return nodeDescriptionFragment;
+    }
+
+    return this.props.node.description;
+  };
+
   handleClose = () => {
     this.props.onCloseOverlayPropertyTable();
   };
 
+  handleSeeFull = () => {
+    this.props.onOpenMatchedNode();
```

Comment:
this function name and the name on line 49 are a bit confusing to me. - I'm not totally sure what `handleSeeFull` and `handleSeeOnlyMatched` should be doing.